components/map/MapCamera.tsx

```diff
@@ -42,6 +42,8 @@ const MapCamera = forwardRef<Camera, Props>(({ cameraZoom, flyToCenter, setFlyTo
     } else if (!isInitialized) {
       setIsInitialized(true)
       setLocalFlyToCenter(MAP_CENTER)
+    } else if (!flyToCenter) {
+      setLocalFlyToCenter(null)
```

Comment:
```suggestion
      setLocalFlyToCenter(undefined)
```
If you set it to undefined you don't have to change the state types and the defaults below
although I do see how setting something to null might seem better  Idon't thin kthere's a real diferrence in this case